lib/grp.gi

```diff
@@ -1706,8 +1706,7 @@ local cheap,nolattice,intersize,attr,kill,i,flags,sup,sub,l;
   # finally kill superseded ones (by replacing with last, which possibly was
   # just added)
   for i in Reversed(Set(kill)) do
-    attr[i]:=attr[Length(attr)];
```

Comment:
This might change behaviour, previously if i==attr, this would mean attr got removed, but now it will stay. This might be OK, because of Reversed(Set(kill)), but I wanted to flag it so it was thought about.

Reply:
Thank you, always a good idea to double check, I appreciate it! I'll assume you meant "if i==Length(attr)", please correct me if I got that wrong.

In the old code and in this situation, the former line 1709 effectively does nothing, but then the `Unbind` in the next line still removes that entry.

While indeed with the new code would end up retaining the entry. I see no reason why this would be correct here -- I simply screwed up :-(.

I'll revert this part of the changes. Thanks for catching it!